packages/cml/src/replaceCmlPlaceholders/replaceCmlPlaceholders.ts

```diff
@@ -70,6 +70,12 @@ const replacePropValues = (mdx: string, values: Record<string, string> = {}): st
           newValue = flatObject(newValue);
         }
         break;
+      case 'BackgroundImage':
+        searchValue = match;
+        if (typeof newValue === 'object') {
+          newValue = flatObject(newValue);
+        }
+        break;
```

Comment:
Should we move the `BackgroundImage` case to be in the same block as `BoxProps` and `FlexProps`?

Reply:
Done!